packages/reactive-element/src/reactive-element.ts

```diff
@@ -762,6 +763,11 @@ export abstract class ReactiveElement
     // finalize any superclasses
     const superCtor = Object.getPrototypeOf(this) as typeof ReactiveElement;
     superCtor.finalize();
+    // Create own set of initializers for this class if any exist on the
+    // superclass and copy them down.
+    if (superCtor._initializers) {
+      this._initializers = [...superCtor._initializers];
```

Comment:
🏌️‍♂️: `this._initializers = [...superCtor._initializers ?? []];`

Reply:
Done for small perf benefit. Added comment.